src/protocol.cc

```diff
@@ -175,6 +175,11 @@ namespace kickcat
     {
         std::stringstream os;
 
+        os << "PDI Operational:  " << std::to_string(dl_status.PDI_op) << " \n";
```

Comment:
not sure that to_string is needed here since this is already a stream that should do the conversion :thinking:

Reply:
is that true for all the prints here?

Reply:
probably... to be tested though.

Reply:
Ok sooo we can remove the `to_string` to all variables that are not a `uint8_t`, but in the case a variable is a `uint8_t` the stringstream parses it as a a char and it breaks the print :).

Reply:
OK